docs/en/install-upgrade/installing-stack-demo-self.asciidoc

```diff
@@ -0,0 +1,713 @@
+// for testing:
+:version: 8.11.1
+
+[[installing-stack-demo-self]]
+=== Tutorial 1: Installing a self-managed {stack}
+
+This tutorial demonstrates how to install and configure the {stack} in a self-managed environment. Following these steps, you'll set up a three node {es} cluster, with {kib}, {fleet-server}, and {agent}, each on separate hosts. The {agent} will be configured with the System integration, enabling it to gather local system logs and metrics and deliver them into the {es} cluster. Finally, you'll learn how to view the system data in {kib}.
+
+It should take between one and two hours to complete these steps.
+
+* <<install-stack-self-prereqs>>
+* <<install-stack-self-overview>>
+* <<install-stack-self-elasticsearch-first>>
+* <<install-stack-self-elasticsearch-config>>
+* <<install-stack-self-elasticsearch-start>>
+* <<install-stack-self-elasticsearch-second>>
+* <<install-stack-self-elasticsearch-third>>
+* <<install-stack-self-kibana>>
+* <<install-stack-self-fleet-server>>
+* <<install-stack-self-elastic-agent>>
+* <<install-stack-self-view-data>>
+* <<install-stack-self-next-steps>>
+
+[IMPORTANT] 
+==== 
+If you're using these steps to configure a production cluster that uses trusted CA-signed certificates for secure communications, after completing Step 6 to install {kib} we recommend jumping directly to <<install-stack-demo-secure>>.
+
+The second tutorial includes steps to configure security across the {stack}, and then to set up {fleet-server} and {agent} with SSL certificates enabled.
+====
+
+[discrete]
+[[install-stack-self-prereqs]]
+== Prerequisites and assumptions
+
+To get started, you'll need the following:
+
+* A set of virtual or physical hosts on which to install each stack component. 
+* On each host, a super user account with `sudo` privileges.
+
+The examples in this guide use RPM packages to install the {stack} components on hosts running Red Hat Enterprise Linux 8. The steps for other install methods and operating systems are similar, and can be found in the documentation linked from each section.
+
+Special considerations such as firewalls and proxy servers are not covered here.
```

Comment:
I think we can put a note here to reference the diagram (now that we have it) at the top for the basic ports and protocols necessary for the installation to work.